packages/types/src/amount.ts

```diff
@@ -77,9 +77,17 @@ export const formatNumber = (number: number, options: FormatOptions): string =>
     : parseFloat(number.toFixed(precision)).toString();
 };
 
-export const formatAmount = (amount: Amount, exponent = 0) =>
+export const formatAmount = ({
+  amount,
+  exponent = 0,
+  withCommas = false,
+}: {
+  amount: Amount;
+  exponent?: number;
+  withCommas?: boolean;
+}) =>
   fromBaseUnitAmount(amount, exponent)
-    .toFormat(6)
+    .toFormat(6, { decimalSeparator: '.', groupSeparator: withCommas ? ',' : '', groupSize: 3 })
     .replace(/(\.\d*?[1-9])0+$|\.0*$/, '$1');
```

Comment:
Before, this was defaulting to returning a string with commas. Seeing that this amount is manipulated in some places later, I think it's safer for consumers to request commas if desired.